packages/runed/src/lib/test/util.svelte.ts

```diff
@@ -29,3 +29,7 @@ export function focus(node: HTMLElement | null | undefined) {
 		flushSync(() => node.focus());
 	}
 }
+
+export function delay(timeout = 0) {
```

Comment:
I believe we have a `sleep` util already that does this exact thing!

Reply:
I made this because I saw no such thing being used in the existing unit tests. If it exists, I can remove this one.  It would be nice, then, if the unit tests were updated at some point to use this sleep function instead of what's being done now, for consistency.

I'll look that sleep function up.

Reply:
Found it.  Can I improve it by making the parameter optional with a value of zero?  As in:

```diff
- export async function sleep(ms: number): Promise<void> {
+ export async function sleep(ms = 0): Promise<void> {
    ...
}
```

Reply:
Yes 😃